src/labels/CustomLabels.labels

```diff
@@ -2929,6 +2929,22 @@
         <shortDescription>Zipcode Verification Limit</shortDescription>
         <value>You cannot verify more than 100 zipcodes at a time.</value>
     </labels>
+    <labels>
+        <fullName>accordionSection_ToggleInstructionsWhenClosed</fullName>
+        <categories>accordionSection</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Accordion instruction when it's closed</shortDescription>
+        <value>Click to toggle the section. This section is not open.</value>
```

Comment:
I recommend we simplify this text to  "Click to expand this section." or "Click to open this section.".